src/lib/markbind/index.js

```diff
@@ -0,0 +1,13 @@
+const htmlparser2patch = require('./src/patches/htmlparser2');
+const markdownItEscapeSpecialTags = require('./src/lib/markdown-it/markdown-it-escape-special-tags');
+const Parser = require('./src/Parser');
+
+function ignoreTags(tagsToIgnore) {
+  htmlparser2patch.injectIgnoreTags(tagsToIgnore);
+  markdownItEscapeSpecialTags.injectTags(tagsToIgnore);
+}
+
+module.exports = {
```

Comment:
Is there a rationale for only exporting the following? (and not other things like `errors`)

*I'm fine with the cleanup as it is though, since we're going to move all the other things inside `core` and change the import / paths again

Reply:
Abstraction, since the caller (CLI) shouldn't need to know how ignoring is done nor access those modules directly.